src/Till.js

```diff
@@ -0,0 +1,50 @@
+var Till = function() {
+  this.detailsList = hipsterCoffeeDetails;
+  this.orderedItems = [];
+  this.subTotal = 0;
+  this.totalCalculator = new TotalCalculator(this.detailsList, this.orderedItems, this.subTotal);
+};
+
+Till.prototype.retrievePrice = function(item) {
+  return this.detailsList[0].prices[0][item];
```

Comment:
I can totally get why you've done this, because I wrote this test. It's a bit of a warning sign if you have to explictily refer to array positions all over your code - either you get them when you load the data OR you have another object specifically to destructure the incoming data